### DISCUSSION TITLE
Option to download ".preview.png" with model

### DISCUSSION THREAD

**Discussion by HyperAniki** [civitai/civitai#232] [Ideas]
Dear Civitai Team,

I hope this suggestion finds you well. I have been using the AUTOMATIC1111 WebUI to run Stable Diffusion Models from civitai.com and I noticed that there is a button that "Shows extra networks" such as TI, Hypernetwork, LORA, and Checkpoints and also displays a thumbnail with the model name.

I would like to suggest adding an option that would allow us to download both the model and the corresponding .preview.png image with the same name as the model. For example, if the model is named "model.safetensors", the associated .preview.png image should be named "model.preview.png". The .preview.png could be the first picture that is shown on the corresponding civitai.com model page. This way, the AUTOMATIC1111 WebUI can display the model and its preview without the hassle of multiple clicks and typing to download and rename the image file.

Having the option to download both the model and its preview image in one click would greatly improve the user experience and save time.

Thank you for considering this suggestion.

Best regards,
A Civitai User

**Comment by Raivshard** [comment]
Yes. This. But when it's downloaded, make it drop the "preview" from the filename, since the name needs to match the model for Automatic1111 to recognize it. Either that or have the first image in the list be automatically downloaded -- or even better, have whichever image is currently first in the user's view be the one downloaded.

**Reply by Trahloc** [reply]
> make it drop the "preview" from the filename, since the name needs to match the model for Automatic1111 to recognize it.

Incorrect, both model.png and model.preview.png work unless you're not using a recent version.  If a model.png exists it takes precedence over model.preview.png though.  For textual inversions / embeddings it *must* be a ti.preview.png as there is a TI standard that can embed itself directly into a .png file and will throw console errors if you put a normal ti.png file in there and cause unnecessary delays to your system.

edit -- and it has to be .png a .jpg or .jfif doesn't work. Even if just take a jpeg file and simple rename it .png it'll work.

**Comment by dennis-gonzales** [comment]
+1

**Comment by xDarkAgentx** [comment]
In a way this seems like an accountability feature more than anything? Make someone post models that **are actual examples** of the model they made

**Reply by Trahloc** [reply]
Yeah, especially if it's an image that can be reproduced by end users since it's done by Civitai instead of whatever magic the person does on their model to look amazing while mixing it with your own it comes out looking like some Lovecraftian horror.
